tests/atomistics/structure/test_atoms.py

```diff
@@ -666,6 +670,17 @@ def test_occupy_lattice(self):
         orig_basis.occupy_lattice(H=O_indices[0])
         self.assertEqual(orig_basis.get_chemical_formula(), "Cl31HNa32")
 
+
+    def test_get_majority_species(self):
+        basis = Atoms(symbols=4*['Fe'], positions=np.random.random((4, 3)), cell=np.eye(3))
```

Comment:
Btw you can also initialize like

`Atoms(symbols='Fe4', positions=np.random.random((4, 3)), cell=np.eye(3))`